### DISCUSSION TITLE
Replace the term "Security Key" with "Passkey"

### DISCUSSION THREAD

**Discussion by tksst** [keycloak/keycloak#25500] [Ideas]
Hello everyone,

I would like to suggest replacing the term "Security Key" with "Passkey" in the UI. I believe that "Passkey" is a more user-friendly term that is easier to understand for non-technical users. "Security Key" gives the impression that only USB connected security keys like Yubikey can be used. In fact, with Keycloak's WebAuthen passwordless authenticator, you can also use a Passkey in a PC's TPM or a synced Passkey.

![image](https://github.com/keycloak/keycloak/assets/2692570/60bd0c81-7d23-47a5-95ad-d07a8962e8ea)

![image](https://github.com/keycloak/keycloak/assets/2692570/3a4b6457-d2cb-49b1-8a71-6d1d5c6a305b)

**Comment by darius-m** [comment]
I doubt either wording is familiar for a non-technical person, and "passkey" also implies the specific passkey implementation of WebAuthn, so it's not much better in that regard. Maybe saying "Use your security key or passkey for sign in." in the description would be clearer.
